backend/entities/BuildingFiscal.cs

```diff
@@ -26,6 +26,12 @@ public class BuildingFiscal : BaseEntity
         /// <value></value>
         public int FiscalYear { get; set; }
 
+        /// <summary>
+        /// get/set - The effective date of this fiscal value
+        /// </summary>
+        /// <value></value>
+        public DateTime EffectiveDate { get; set; }
```

Comment:
As this isn't nullable, how will existing data be handled in the migration?

Reply:
Existing rows would be assigned 01/01/0001, so will probably need to make nullable